dev/AppNotifications/AppNotificationUtility.h

```diff
@@ -41,6 +41,16 @@ namespace Microsoft::Windows::AppNotifications::Helpers
         return (str == nullptr || wcslen(str) == 0) ? wil::unique_hstring() : wil::make_unique_string<wil::unique_hstring>(str);
     }
 
+    inline void RethrowIf(std::exception_ptr eptr) try
+    {
+        if (eptr) std::rethrow_exception(eptr);
```

Comment:
nit:

```
if (...)
{
// some code
}
```

Reply:
I know it's just one line, but that's the format we follow.

Reply:
Hopefully this code will go away once we've decided between option 1 and 2. Otherwise I'll fix it

Reply:
Code is gone